liquibase-integration-tests/src/test/java/liquibase/dbtest/pgsql/PostgreSQLIntegrationTest.java

```diff
@@ -237,4 +243,25 @@ public void correctlyHandlesAutoIncrementSequences() throws Exception {
         assert !snapshot.get(new Table(null, null, "owned_by_table")).getColumn("id").isAutoIncrement();
 
     }
+
+    @Test
+    public void testGeneratedColumn() throws Exception {
+        assumeNotNull(getDatabase());
+        assumeTrue(getDatabase().getDatabaseMajorVersion() >= 12);
+        clearDatabase();
+        String textToTest = "GENERATED ALWAYS AS (height_cm / 2.54) STORED";
```

Comment:
I saw this in the Postgres docs, too. I thought it was a good example of a concrete application for the GENERATED columns. 

Thanks for adding the test!